pkg/service/restore/helper_integration_test.go

```diff
@@ -356,7 +356,7 @@ func dropNonSuperUsers(t *testing.T, s gocqlx.Session) {
 
 func createUser(t *testing.T, s gocqlx.Session, user, pass string) {
 	if err := s.ExecStmt(fmt.Sprintf("CREATE USER '%s' WITH PASSWORD '%s'", user, pass)); err != nil {
-		t.Fatal(errors.Wrapf(err, "create user %s with pass %s", user, pass))
+		t.Logf("create user %s with pass %s, err = {%v}", user, pass, err)
```

Comment:
Why shouldn't it fail the test? This user should be used for restore, so if it fails, the restore should fail as well.

Reply:
it should, revert